development/sourcemap-validator.js

```diff
@@ -20,7 +20,15 @@ start().catch((error) => {
 
 
 async function start () {
-  const targetFiles = [`inpage.js`, `contentscript.js`, `ui.js`, `background.js`]
+  const targetFiles = [
+    `background.js`,
+    // `bg-libs`, skipped because source maps are invalid due to browserify bug: https://github.com/browserify/browserify/issues/1971
+    // `contentscript.js`, skipped because the injected inpage script is resulting in invalid source maps
```

Comment:
Ah, I should clarify here: I suspect that the _validator_ is in the wrong here, because it's seeing the string `new Error` in the `inpage.js` string-literal and mistakenly trying to map it onto a `new Error` in the source code.